### PR TITLE
Add support for COMMENT ON COLUMN statement

### LINKED ISSUE #2242: Add support for comment on column

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #2242

### CODE REVIEW

**Comment by martint** [comment]
Syntax looks good. It's consistent with COMMENT ON TABLE  and the eventual direction we're going with this. See (https://github.com/prestosql/presto/pull/200#discussion_r257038416 for previous discussions)

File: presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java [hunk 1]
```diff
@@ -1526,6 +1526,14 @@ public void testCommentTable()
         assertStatement("COMMENT ON TABLE a IS NULL", new Comment(Comment.Type.TABLE, QualifiedName.of("a"), Optional.empty()));
     }
 
+    @Test
+    public void testCommentColumn()
+    {
+        assertStatement("COMMENT ON COLUMN a.b IS 'test'", new Comment(Comment.Type.COLUMN, QualifiedName.of("a", "b"), Optional.of("test")));
```
Comment (martint) [inline presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java:1532]: Add tests for other forms of column names: E.g., `catalog.schema.table.column`, `schema.table.column`, `column`, etc.

File: presto-main/src/main/java/io/prestosql/execution/CommentTask.java [hunk 2]
```diff
@@ -58,6 +61,23 @@ public String getName()
 
             metadata.setTableComment(session, tableHandle.get(), statement.getComment());
         }
+        else if (statement.getType() == Comment.Type.COLUMN) {
+            QualifiedObjectName tableName = createQualifiedObjectName(session, statement, statement.getName().getPrefix().get());
+            Optional<TableHandle> tableHandle = metadata.getTableHandle(session, tableName);
```
Comment (martint) [inline presto-main/src/main/java/io/prestosql/execution/CommentTask.java:66]: This needs to check if the name has a single part. E.g., if the user enters `COMMENT ON foo IS ...`

**Comment by ebyhr** [comment]
Updated.

**Pull request merged**